src/quickstart/provider/aws/boot.md

```diff
@@ -1,263 +1,364 @@
-authentication and endpoint information.
+This is a critical component of setting up clusters. If you are experiencing
+problems with the Kubernetes API not coming up, chances are something is wrong
+with the clusters DNS.
 
+**Please DO NOT MOVE ON until you have validated your NS records!**
+
+
+## Cluster State storage
+
+In order to store the state of your cluster, and the representation of your
+cluster, we need to create a dedicated S3 bucket for `kops` to use.  This
+bucket will become the source of truth for our cluster configuration.  In
+this guide we'll call this bucket `example-com-state-store`, but you should
+add a custom prefix as bucket names need to be unique.
+
+We recommend keeping the creation of this bucket confined to us-east-1,
+otherwise more work will be required.
+
+```bash
+aws s3api create-bucket --bucket prefix-example-com-state-store --region us-east-1
 ```
-cluster "aws_kubernetes" set.
-user "aws_kubernetes" set.
-context "aws_kubernetes" set.
-switched to context "aws_kubernetes".
-user "aws_kubernetes-basic-auth" set.
-Wrote config for aws_kubernetes to /Users/jhansen/.kube/config
+
+Note: We **STRONGLY** recommend versioning your S3 bucket in case you ever need
+to revert or recover a previous state store.
+
+```bash
+aws s3api put-bucket-versioning --bucket prefix-example-com-state-store  --versioning-configuration Status=Enabled
 ```
 
-Up next, worker nodes are provisioned by an auto-scaling group, and we wait for those nodes to come up.
 
+## Creating your first cluster
+
+#### Prepare local environment
+
+We're ready to start creating our first cluster!  Let's first setup a few
+environment variables to make this process easier.
+
+```bash
+export NAME=myfirstcluster.example.com
+export KOPS_STATE_STORE=s3://prefix-example-com-state-store
 ```
-Creating minion configuration
-Creating autoscaling group
- 0 minions started; waiting
- 0 minions started; waiting
- 0 minions started; waiting
- 0 minions started; waiting
- 1 minions started; waiting
- 1 minions started; waiting
- 1 minions started; waiting
- 2 minions started; ready
-Waiting for cluster initialization.
-
-  This will continually check to see if the API for kubernetes is reachable.
-  This might loop forever if there was some uncaught error during start
-  up.
-Waiting for cluster initialization.
-
-  This will continually check to see if the API for kubernetes is reachable.
-  This might loop forever if there was some uncaught error during start
-  up.
-
-.................................................................................................................Kubernetes cluster created.
-Sanity checking cluster...
-Attempt 1 to check Docker on node @ 52.53.207.230 ...working
-Attempt 1 to check Docker on node @ 52.53.172.73 ...working
+
+Note: You don’t have to use environmental variables here. You can always define
+the values using the –name and –state flags later.
```

Comment:
let's wrap -name and -state in backticks. We usually wrap CLI option flags, source code, anything terminal-related with backticks.